app/_src/gateway/kong-enterprise/cp-outage-handling-faq.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Control Plane Outage Management FAQ
+badge: enterprise
+content_type: Reference
+---
+
+Starting in {{site.base_gateway}} version 3.2, {{site.base_gateway}} can be configured to support configuring new data planes in the event of a control plane outage. This document serves as an informational set of guidelines for that feature. For instructions on setting this up, read the [data plane resilience documentation](/gateway/latest/kong-enterprise/cp-outage-handling) doc.
+
+
+## How it works
+
+When the cluster adds new data plane nodes, the control plane uses a configuration file to provision those nodes. If the control plane experiences an outage, the data plane is unable to be provisioned, and will silently fail until they can establish a connection with the control plane. After configuring {{site.base_gateway}} to manage the addition of new data plane nodes in the event of a control plane outage new nodes will no longer silently fail after not being able to establish a connection with the control plane and are configured by reading the configuration file that is located in the designated S3 compatible storage volume. 
+
+By designating a dedicated backup node, any changes to the configuration file are pushed to the S3 compatible storage. Any new data plane nodes read the configuration file from the S3 compatible storage volume and consume the new configuration changes. 
+
+If the data plane and control plane are both configured to `export` a configuration, assuming that they are both configured with the right level of authentication, the data plane will export the configuration to the bucket.
```

Comment:
By "latter writes win" I mean "later writes win". Sorry for my misleading comment. It's possible that DP writes to the stroage faster than the CP, and the CP overwrite the config. Anyway, we could just tell the user to not configure it this way.